.travis.yml

```diff
@@ -38,3 +38,5 @@ notifications:
       - antonbachin@yahoo.com
     on_success: always
     on_failure: always
+
+sudo: required
```

Comment:
ok, I'll remove it. somehow I did need this to get travis to build & run the tests on my fork of the lwt repo...